Gemfile

```diff
@@ -40,6 +40,8 @@ group :development, :test do
   gem 'capybara', '~> 2.13.0'
   gem 'selenium-webdriver'
   gem 'rubocop', '~> 0.75.0', require: false
+  gem 'dotenv-rails'
```

Comment:
I think that using [pessimistic lock](https://thoughtbot.com/blog/rubys-pessimistic-operator) on your gems versions is a good idea, It's a little extra point you add to the stability of your dependencies within the project.

Reply:
Ok :)